docs_source_files/content/webdriver/support_classes.en.md

```diff
@@ -1,5 +1,76 @@
 ---
 title: "Support classes"
-weight: 5
+weight: 4
 ---
 
+Webdriver support classes are provided to make updating and maintaining your code easier. They provide a nice abstraction to make modelling HTML element(s) as domain objects easier, 
+also providing helper methods to make using such objects easy to reason about.
+We will learn about Locator Strategies, Events, PageFactory, LoadableComponent, ThreadGuard etc.
+
+Let's Start:
+
+## **ThreadGuard**
+ThreadGuard checks that a driver is called only from the same thread that created it. 
+Threading issues especially when running tests in Parallel may have mysterious and hard to diagnose errors. 
+Using this wrapper prevents this category of errors and will raise an exception when it happens. 
+
+The following example simulate a clash of threads:
```

Comment:
Please only add Java code, no need to use the shortcode.

So, just use

```
    ```java 
    Here Java Code
    ```
```

Reply:
Done